tests/model_fields_/test_objectidfield.py

```diff
@@ -0,0 +1,41 @@
+from bson import ObjectId
+from django.core import exceptions
+from django.test import SimpleTestCase
+
+from django_mongodb.fields import ObjectIdField
+
+
+class MethodTests(SimpleTestCase):
+    def test_deconstruct(self):
+        field = ObjectIdField()
+        name, path, args, kwargs = field.deconstruct()
+        self.assertEqual(path, "django_mongodb.fields.ObjectIdField")
+        self.assertEqual(args, [])
+        self.assertEqual(kwargs, {})
+
+    def test_get_internal_type(self):
+        f = ObjectIdField()
+        self.assertEqual(f.get_internal_type(), "ObjectIdField")
+
+    def test_to_python(self):
+        f = ObjectIdField()
+        expected = ObjectId("1" * 24)
+        for value in ["1" * 24, ObjectId("1" * 24)]:
```

Comment:
I'd prefer separate tests or a string of `to_python()` calls without the loops and subTest, e.g. 
```
    def test_to_python(self):
        self.assertIsNone(models.UUIDField().to_python(None))

    def test_to_python_int_values(self):
        self.assertEqual(
            models.UUIDField().to_python(0),
            uuid.UUID("00000000-0000-0000-0000-000000000000"),
        )
        # Works for integers less than 128 bits.
        self.assertEqual(
            models.UUIDField().to_python((2**128) - 1),
            uuid.UUID("ffffffff-ffff-ffff-ffff-ffffffffffff"),
        )
```
I think the current way is too hard on the eyes.